script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -143,23 +143,23 @@ But it's true what they say about not being good in the morning, right?"
 Senpai chuckles.
 I am taken aback by her gesture.
 <Page39>
-"Look. It's me, it's me. I met you just last week, have you forgotten me already?"
+"Look. It's me, it's me. I met you just last week, have you forgotten about me already?"
 <Page40>
 "Huh, ......Ciel-senpai......?"
 In an instant, my memory is jolted.
 Of course she looks familiar. I've met this person several times at school.
 Actually, there's probably no one in our school who doesn't know Ciel-senpai.
 <Page41>
-At some point, she started to be called "The Almighty Godfather Senpai".
-If there are first years who are troubled, she would counsel and help them feel better.
-If there are second years who are lost, she would guide them herself.
-If there are students in trouble with the third years, she would stop them from bullying their underclassmen.
+At some point, she started to get called "The Almighty Godfather Senpai".
+If there are first years who are troubled, she will counsel and help them feel better.
+If there are second years who are lost, she will guide them herself.
+If there are students in trouble with the third years, she will stop them from bullying their underclassmen.
 <Page42>
-Some students even call her the real student council president, because she is more dependable than the teachers and the school council.
-It goes without saying that she helped me out last week.
+Some students even call her the real student council president, because she is more dependable than the teachers and the school council themselves.
+It goes without saying that she has helped me out last week.
```

Comment:
Helping me out last week should be past tense (drop the 'has')

Reply:
Actually, on rereading it's a bit weird that there's a "last week" in here? It might be obvious that she helped him out _in the past_, but last week is pretty specific. Rereading the source - 

```
言うまでもなく、俺も先週、ちょっとした事で助けてもらったはずだ
言うまでもなく (ゆうまでもなく) (exp) obviously; as we all know; needless to say; of course; it goes without saying; 
先週 (せんしゅう) (n-adv,n-t) last week; the week before; (P); 
```

So he does say last week, but "goes without saying she helped me last week" implies some prior knowledge of the reader about _the event_, which we don't have here - maybe "It goes without saying that _when I ran into trouble_ last week, she lent me a hand / helped me out", which explains away the event as secondary to the action of helping out. Again, tough to put into concrete language terms, sorry.